hack/scripts/copylogs

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+LOG_DIR=logs
+
+mkdir -p $LOG_DIR
+
+for m in `vagrant status | grep running | awk '{print $1}'`; do 
+  echo Getting logs for $m
+  vagrant ssh $m -- sudo /tmp/gatherlogs &> /dev/null
```

Comment:
Read from stdin so we don't have to include the `gatherlogs` script at provision time?

```
vagrant ssh $m -- sudo /bin/bash < gatherlogs
```

Reply:
@rphillips was this addressed?

Reply:
removed the vagrant integration.